### DISCUSSION TITLE
Repository access issue for students joining assignment

### DISCUSSION THREAD

**Discussion by soroberts** [community/community#138930] [GitHub Education]
### Hi everyone,

I run two labs back-to-back on Tuesdays. Students from the 1:20 pm lab were able to access the lab, but students in the 2:50 lab are not. They get a "Repository access issue" error. It is the same exact assignment, all students are on the same student roster. 

I was able to replicate the error with a second assignment. Most students accepted this other assignment before today, but one student had not. The student who had not yet accepted the second assignment had the same repository access issue as all of my other students are having on the lab for today.

**Comment by djmandell** [comment]
I can confirm the same problem.  It started roughly... 40 minutes ago maybe?

**Comment by msoehasker** [comment]
For me, the issue was inconsistent. Some students had the above problem, others did not. I was able to fix the problem by going to the student's repo settings and adding the student explicitly with write access (as the student was not even listed as a collaborator).

**Reply by Ujoa** [reply]
yeah thats what ive been doing manually. its not sustainable when an assignment opens up and 300+ try to accept tho

**Comment by chrissphinx** [comment]
bumping this one too

**Comment by svlasnik-uno** [comment]
Same issue. Any resolution yet?
